ovos_plugin_manager/pipeline.py

```
@@ -1,41 +1,56 @@
 from ovos_bus_client.client import MessageBusClient
+from ovos_bus_client.message import Message
 from ovos_config import Configuration
 from ovos_utils.fakebus import FakeBus
-from ovos_utils.log import log_deprecation
 
-from ovos_plugin_manager.templates.pipeline import ConfidenceMatcherPipeline, PipelineStageMatcher, PipelinePlugin
-from ovos_plugin_manager.utils import PluginTypes
+from ovos_plugin_manager.templates.pipeline import (
+    PipelineMatch,
+    ConfidenceMatcherPipeline,
+    PipelineStageMatcher,
```

Ruff (0.11.9)

`ovos_plugin_manager.templates.pipeline.PipelineStageMatcher` imported but unused

Remove unused import: `ovos_plugin_manager.templates.pipeline.PipelineStageMatcher`

(F401)